### ISSUE TITLE
Do you even review and execute this code?

### ISSUE THREAD

**Issue by dexblack** [MicrosoftDocs/cpp-docs#1968]
Platform::String has no char* constructor.
so the code passing a char* orig = "Hello, World!"; to Platform::String(orig);  is just wrong.
wchar_t *orig = _T("Hello, World!");  is dependent on build conditions and ought to be written:
wchar_t *orig = L"Hello, World!"; unfortunately the Interop doesn't honor "const" or readonly parameters so there's no point trying to write const correct code in that realm.

How are beginner programmers supposed to get this basic stuff right when you provide such utterly incorrect code?
If I looked harder I'm certain to find more mistakes but I'm too disgusted to continue.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bd575689-da23-4f4e-3d7c-63ce6ba63a31
* Version Independent ID: 8656c596-5e50-4b20-c828-849aec8ac28e
* Content: [How to: Convert Between Various String Types](https://docs.microsoft.com/en-us/cpp/text/how-to-convert-between-various-string-types?view=vs-2019#feedback)
* Content Source: [docs/text/how-to-convert-between-various-string-types.md](https://github.com/Microsoft/cpp-docs/blob/master/docs/text/how-to-convert-between-various-string-types.md)
* Product: **visual-cpp**
* Technology: **cpp-windows**
* GitHub Login: @TylerMSFT
* Microsoft Alias: **twhitney**

**Comment by TylerMSFT** [comment]
Yes, this topic is pretty old and has problems. I'll put it on my list to fix.

**Comment by TylerMSFT** [comment]
We are in the process of moving away from GitHub issues for C++ documentation (we still accept pull requests). For details, please see [Announcing a new way to give feedback on Microsoft Learn](https://techcommunity.microsoft.com/t5/microsoft-learn-blog/announcing-a-new-way-to-give-feedback-on-microsoft-learn/ba-p/4027635) This issue will be closed after it is copied to an internal tracking system.
